src/models/BasicModel.js

```diff
@@ -40,6 +40,7 @@ class BasicModel extends Model {
     donationCounters = [],
     token,
     createdAt,
+    updatedAt,
```

Comment:
Is it necessary to add this?

Reply:
Please test without adding updatedAt here and other places. Just keep it in query.

Reply:
It's not necessary. I added it to check the list dates.